cglib/src/test/java/net/sf/cglib/classloader/FilteringClassLoader.java

```diff
@@ -0,0 +1,119 @@
+package net.sf.cglib.classloader;
```

Comment:
Is copyright header missing here?

Reply:
This particular code wasn't actually developed by Apache Foundation, so it's technically incorrect to include its copyright header here. The cglib process itself has license and notice files in the project structure, so there is no need to include copyright header in every file, I assume. However, if there is any defined file-header template for the project - it can be applied, but I haven't found any.